suites/nautilus/ansible/tier-0.yaml

```diff
@@ -0,0 +1,72 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/nautilus/ansible/tier-0.yaml
+# Smoke test cases for
+#    - Install RHCS4 with lvm scenario
+#
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      config:
+        is_production: true
```

Comment:
Can we remove this key/value? The primary purpose of this suite is to test against the development build hence we could opt to use the internal credentials